projects/ngqp/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@ngqp/core",
-    "version": "14.1.0",
+    "version": "15.0.0",
```

Comment:
Please undo this change, this is done during release. Also, move the

> BREAKING CHANGE: due to partial ivy compilation it's not supported by angular before 11.1 so bump version to 15

comment into the commit message, because that is where conventional commits need it. But leave out the "version 15" there, that is decided during release. Just an explanation of why it is breaking: the dependencies bump. 🙂